src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -57,7 +57,98 @@ The module connects to the PS3 controller using bluetooth (or USB) and reacts ba
 | SQUARE                    | Toggles locking of the head.               |
 | R2                        | Right kick.                                |
 
-To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and run the binary. Plug the PS3 controller into the robot using a USB cable. When the controller is connected to the robot, it will be paired, and you can remove the cable.
+To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and install it on the robot. Ensure the PS3 controller is paired and connected to the right robot. Then run the `./ps3walk` on the robot to begin controlling it.
+
+#### Setup
+
+1. Install `bluez`, `bluez-utils`, and `bluez-plugins` if they have not already been installed:
+   ```
+   pacman -S --needed bluez bluez-utils bluez-plugins
+   ```
+2. Restart the system:
+   ```
+   reboot now
+   ```
+   This is only necessary if the packages were not previously installed.
+3. Start the bluetooth service:
+   ```
+   systemctl start bluetooth.service
+   ```
+
+<details>
+<summary> Start bluetooth automatically </summary>
+
+Bluetooth can be powered on automatically when the system starts. This enables the controllers to
+connect without having to run `systemctl start bluetooth.service` and `bluetoothctl power on`. This
+is not enable by default because we're currently unsure whether wifi and bluetooth are interferring
```

Comment:
Not sure if you found a different issue, but wheen using the other robot the wifi and bluetooth work fine. The robot you were origionally using had a dodgy connector for bluetooth/wifi. The nuc has been swapped out now for a spare.

Reply:
Yeah, I wrote this before someone figured that out.

Reply:
Quick question. What would be an appropriate thing to say here? The bluetooth service isn't enabled by default in the system installation, but we've enabled it on some of the robots. Should bluetooth be turned on for all of them in the install script?

Reply:
Yes, I think it should be set up for all robots when they're initially set up. It would still be good to mention it in this page in case something goes wrong and someone needs to figure out how to do it.